modules/custom/openy_activity_finder/modules/openy_paragraph_activity_finder_search/openy_paragraph_activity_finder_search.install

```diff
@@ -17,3 +17,19 @@ function openy_paragraph_activity_finder_search_update_8002() {
     'core.entity_form_display.paragraph.activity_finder_search.default'
   ]);
 }
+
+/**
+ * Update configs.
+ */
+function openy_paragraph_activity_finder_search_update_8003() {
+  // We exported this configs to sync, so no need to import this in update.
+  return;
```

Comment:
Why are we adding this update function to Open Y?

Reply:
I believe some YMCA already using AF.
This is hook update.
@duozersk ?

Reply:
Yes, many YMCA are using AF, lets keep it (return is ygbw specific, I'm going to remove it)

Reply:
It has a "return;" statement as the first executable line... so apparently it does nothing.

Reply:
It has not anymore